js/main.js

```diff
@@ -69,6 +70,8 @@ document.addEventListener('DOMContentLoaded', function (event) {
     const $data = renderEntry(data.entries[i]);
     $ul.appendChild($data);
   }
+  toggleNoEntries();
+  viewSwap('entry-form');
```

Comment:
Instead of setting `entry-form`, you should be utilizing the `view` property in the `data` object, so the user will see whatever page they were on last